python/cudf/cudf/tests/test_list.py

```diff
@@ -759,3 +759,13 @@ def test_listcol_setitem_retain_dtype():
     # prior to this fix: https://github.com/rapidsai/cudf/pull/10151/
     df2 = df1.copy()
     assert df2["a"].dtype == df["a"].dtype
+
+
+def test_list_astype():
```

Comment:
Should we also compare the leaf values ?

Reply:
Fixed - thanks!